.pre-commit-config.yaml

```diff
@@ -42,7 +42,13 @@ repos:
     rev: v2.2.2
     hooks:
     -   id: codespell
-        types_or: [python, rst, markdown]
+        exclude: |
+            (?x)
+            pandas/tests/io/data/xml
+            |pandas/_libs/ops_dispatch.pyx
+            |pandas/_libs/tslibs/strptime
+            |web/pandas/config.yml
+
```

Comment:
how about expanding `types_or` instead of excluding certain files? `cython` and `c` should probably be in there

Reply:
Thanks for so fast replay!

We should add `cython` and `c`, instead of exclude some files?

Reply:
yup

Reply:
This time?